.ci/install_cri_containerd.sh

```diff
@@ -8,67 +8,70 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -o errtrace
+
+handle_error() {
+	local exit_code="${?}"
+	local line_number="${1:-}"
+	echo "Failed at $line_number: ${BASH_COMMAND}"
+	exit "${exit_code}"
+}
+trap 'handle_error $LINENO' ERR
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 cri_repository="github.com/containerd/cri"
 
 # Flag to do tasks for CI
 CI=${CI:-""}
 
+# shellcheck source=./lib.sh
 source "${script_dir}/lib.sh"
+
+GOOS=$(go env GOOS)
+GOARCH=$(go env GOARCH)
```

Comment:
Again, this looks pretty generic so could go into one of the library scripts we source.